### PR TITLE
Pydantic models refactoring, separating create and update request models

### PR DESCRIPTION
**Pull request by VeronikaHh**
Pydantic models refactoring, separating create and update request models

### CODE REVIEW

File: app/api/customer/models.py [hunk 1]
```diff
@@ -2,15 +2,20 @@
...
+    name: str | None = None
+    email: str | None = None
+    phone_number: str | None = None
+    is_ower: bool | None = None
 
-class CustomerCreateUpdate(SQLModel):
+class CustomerCreate(SQLModel):
     name: str = Field()
     email: str = Field()
     phone_number: str = Field()
```
Comment (taras-hrechukh) [inline app/api/customer/models.py:14]: Let's add here (in this PR as well) uniqoe constraints for this fields (`email` and `phone_number`)
As this changes lie on db level you should also create the migration for that

File: app/api/employee/models.py [hunk 2]
```diff
@@ -2,14 +2,24 @@
...
+    email: str | None = None
+    phone_number: str | None = None
+    address: str | None = None
+    is_admin: bool | None = None
+
+
+class EmployeeCreate(SQLModel):
     name: str = Field()
     email: str = Field()
     phone_number: str = Field()
```
Comment (taras-hrechukh) [inline app/api/employee/models.py:17]: Same thing for this model. Let's make `email` and `phone_number` fields unique.

File: app/api/order/models.py [hunk 3]
```diff
@@ -5,8 +5,8 @@
 
 
 class OrderUpdate(SQLModel):
-    end_date: datetime = Field(default_factory=lambda: datetime.now(timezone.utc) + timedelta(days=90))
-    is_returned: bool = Field(default=False)
+    end_date: datetime | None = Field(default_factory=lambda: datetime.now(timezone.utc) + timedelta(days=90))
+    is_returned: bool | None= Field(default=False)
```
Comment (taras-hrechukh) [inline app/api/order/models.py:9]: For OrderUpdate we don't need this default values. They were used for creation, but on update you should only put like -> `is_returned: bool | None = None` and the same for `end_date`

**Review by taras-hrechukh — COMMENTED** [review]
Left a few comments

File: app/api/order/models.py (hunk 3 above)
Comment (VeronikaHh) [reply app/api/order/models.py]: fixed

**Review by taras-hrechukh — APPROVED** [review]
Great!

**Pull request merged**